Protocol.md

```diff
@@ -732,19 +751,25 @@ RSocket resumption exists only for specific cases. It is not intended to be an 
 1. Resumption is always initiated by the client and either allowed or denied by the server.
 1. Resumption makes no assumptions of application state for delivered frames with respect to atomicity, transactionality, etc. See above.
 
+### Resume Agreement
+
+If communication with resumption support has been successfully established, the decision on whether a particular REQUEST should be resumed is made for every stream individually. To make resumption enabled for a REQUEST, the Responder MUST explicitly set the (R)esume flag in the first response FRAME to the Requester. Once, the Requester received the first FRAME from the Responder and that frame has (R)esume flag set, then the current stream is assumed to be resumable. Otherwise, if the first Responder FRAME does not have (R)esume flag set, the stream is MUST be considered as not resumable. If a Requester has sent a REQUEST frame and then the connection was lost leaving the resumption for that stream in the unknown state, such a stream should wait until the connection is reestablished, and the RESUME | RESUME_OK frame is received. If RESUME | RESUME_OK includes the mentioned stream in the list of resumable streams, then the resumption is assumed to be enabled for that stream. Otherwise, the stream may be terminated with the __REJECTED__ or retransmit frame again.
```

Comment:
Could we please improve the wording of the last sentence?

> Otherwise, the stream may be terminated with the __REJECTED__ or retransmit frame again.

What frame is retransmitted?

Reply:
Non, it should be terminated locally since the remote party decided to discontinue that stream. I will improve wording, had the same that feeling of absence "locally only" words but did not how to keep it short